.travis.yml

```diff
@@ -16,23 +16,18 @@ env:
 matrix:
   fast_finish: true
   include:
-  - php: 5.6
-    env: MAGENTO_VERSION=2.0.12
-  - php: 5.6
-    env: MAGENTO_VERSION=2.0.*
-  - php: 7.0
-    env: MAGENTO_VERSION=2.1.0
-  - php: 7.0
-    env: MAGENTO_VERSION=2.1.*
   - php: 7.1
     env: MAGENTO_VERSION=2.2.0
   - php: 7.1
-    env: MAGENTO_VERSION=2.2.* CODE_QUALITY=true
+    env: MAGENTO_VERSION=2.2.*
   - php: 7.2
     env: MAGENTO_VERSION=2.3.0
   - php: 7.2
     env: MAGENTO_VERSION=2.3.*
-
+  - php: 7.3
+    env: MAGENTO_VERSION=2.3.0 CODE_QUALITY=true
```

Comment:
Not sure what the `CODE_QUALITY` flag does, but should this again be on the `.*` env?

Reply:
The flag will tell this pipeline to run the coding standard tests like phpcs and magento ecg code style checks. See here
https://github.com/firegento/firegento-magesetup2/blob/develop/Test/script/travis_script.sh#L13